learn/react-native/mobile-build-manual.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "React Native - Mobile Manual Build"
+id: "mobile-build-manual"
+sidebar id: "mobile-build-manual"
+---
+---
+
+In the manual build process using React Native Command Line Interface (CLI), you can generate the WaveMaker studio project in the local environment where additional customization and debugging of project code can be performed. The React Native CLI commands are also used to preview the application.
```

Comment:
I think we should mention that WMO is able to build IPA, APK file automatically using AppChef and the manual process outlined in this document is for teams that do not want to use it and want implement a CI/CD pipeline themselves.